repose-aggregator/components/filters/saml-policy-translation-filter/src/main/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilter.scala

```diff
@@ -97,13 +100,25 @@ class SamlPolicyTranslationFilter @Inject()(configurationService: ConfigurationS
   }
 
   /**
-    * Gets the SAMl response from the post encoded body, and decodes it into a plain String.
+    * Gets the SAMl response from the post encoded body, and decodes it.
     *
     * @param request the servlet request that has an encoded body
     * @return the decoded saml response
     * @throws SamlPolicyException if decoding fails
     */
-  def decodeSamlResponse(request: HttpServletRequest): InputStream = ???
+  def decodeSamlResponse(request: HttpServletRequest): InputStream = {
+    try {
+      Option(request.getParameter("SAMLResponse"))
```

Comment:
I would think so, too, but I would have figured it would mention it in the JavaDoc.  Either it's so obvious it would it's not worth mentioning, or it isn't. &nbsp;&nbsp;&nbsp; Guess we'll see.